crates/iota/src/client_commands.rs

```diff
@@ -496,6 +496,8 @@ pub enum IotaClientCommands {
         #[arg(long, short)]
         profile_output: Option<PathBuf>,
     },
+    /// Remove an existing address by its alias or hexadecimal string.
```

Comment:
I think this should mention that it removes the private key from the keystore and that there is no way to recover, if no backup was made

Reply:
https://github.com/iotaledger/iota/pull/7151/commits/94137616d5e36ee9a52362d3ae496ec70214c50a